components/authentication/component-maintainer.yaml

```diff
@@ -33,3 +33,9 @@ rules:
     verbs:
       - get
       - list
+  - apiGroups:
+    - ''
+    resources:
+    - pods/exec
```

Comment:
Just thought about this while working with pprof, `pods/portforward` would be good here too 🙂

Reply:
here you go d89a550